### ISSUE TITLE
Queue configuration files are not cleaned up when a helix queue is deleted

### ISSUE THREAD

**Issue by riarenas** [dotnet/dnceng#1426]
The "configuration" blob storage that holds all the helix queue configuration files still has configuration zips for queues that have already been deleted. 

I'm not sure if pieces of the automation are reading these files, but we should make sure that removing a queue from the yaml definitions also removes the configuration files associated with them.

### Release Note Category
- [ ] Feature changes/additions 
- [ ] Bug fixes
- [ ] Internal Infrastructure Improvements
### Release Note Description

**Comment by missymessa** [comment]
Do we know if there was a reason we held onto the old config files, or was it more of an oversight when cleaning up old stuff?

**Comment by riarenas** [comment]
Nope. No idea. I opened a similar issue for the autoscaler: https://github.com/dotnet/dnceng/issues/1425 so my sense is these are just oversights.
